File: 9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        long long result = 0;
        int original =x;
        if(x<0){
                return false;
        }
        else{
        while(x!=0){
            result=result*10+(x%10);  
            x/=10;
        }
    return result == original;
    }
    }
};

File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
    int reverse = 0;
    int dup = x;
    while(x>0){
        int ld = x%10;
        reverse = (reverse*10) + ld;
        x/=10;
    }
    if(reverse == dup){
        return true;
    }
    return false;
    }
    }
   